services/app-api/serverless.yml

```diff
@@ -197,6 +210,19 @@ functions:
               paths:
                 state: true
                 reportId: true
+  deleteReport:
+    handler: handlers/banners/delete.deleteReport
```

Comment:
I think this path got mixed up, should be `handlers/reports/delete.deleteReport`, right?